app/server/logging/jsondb/src/main/java/io/syndesis/server/logging/jsondb/controller/KubernetesSupport.java

```diff
@@ -66,9 +73,17 @@ public KubernetesSupport(KubernetesClient client) {
     protected void watchLog(String podName, Consumer<InputStream> handler, String sinceTime, Executor executor) throws IOException {
```

Comment:
is this used somewhere else where the changes can alter current behavior?

Reply:
This should add a rest call to get the pod details, but not alter the behavior. In fact, if you try to stream the logs from a pod with multiple containers, without specifying the container name, you get http error 4xx.

I've left the standard behavior when the number of containers is <= 1.

KubernetesSupport is used only in ActivityTracking, so to stream logs from integration pods. Integration pods in standard s2i mode are composed of a single container, so nothing should change.

Reply:
I stream the logs of the first container also because sidecars are usually appended at the end of the list.

Reply:
Cool thanks.